sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -109,6 +116,25 @@ def from_connection_string(cls, connection_string: str, **kwargs: Any) -> "Azure
             id_credential=id_credential,
             **kwargs,
         )
+    
+    @distributed_trace
+    def send_request(self, request: HttpRequest, **kwargs) -> HttpResponse:
```

Comment:
Shouldn't the signature

def send_request(self, request: HttpRequest, *, stream: bool = False, **kwargs) -> HttpResponse:

?

Reply:
Will update.

Reply:
Updated